<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>毛绒检测</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_imageAc"/>
    <addaction name="close_imageAc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="image_processAc"/>
    <addaction name="compute_dimAc"/>
    <addaction name="show_computeAc"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open_imageAc">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="close_imageAc">
   <property name="text">
    <string>关闭图片</string>
   </property>
   <property name="toolTip">
    <string>关闭图片</string>
   </property>
  </action>
  <action name="image_processAc">
   <property name="text">
    <string>预处理</string>
   </property>
  </action>
  <action name="compute_dimAc">
   <property name="text">
    <string>计算直径</string>
   </property>
  </action>
  <action name="show_computeAc">
   <property name="text">
    <string>显示计算直径动画</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
